ReflectometryServer/test_modules/data_mother.py

```diff
@@ -502,7 +506,7 @@ def create_mock_JawsCentrePVWrapper(name, init_position, max_velocity, backlash_
 BENCH_MIN_ANGLE = 0
 BENCH_MAX_ANGLE = 4.8
 
-def get_standard_bench(with_z_position=0, with_angle=ANGLE_OF_BENCH, vertical_mode=False):
+def get_standard_bench(with_z_position=0, with_angle=ANGLE_OF_BENCH, vertical_mode=False, perp_to_floor_angle=90.0):
```

Comment:
This has changed from an `int` to a `float` - does this have the potential to cause any unwanted to behaviour besides the obvious use of more memory? I guess you are gaining precision, but things like comparisons may slow the server down too? I think float comparison is quite expensive if not using a tolerance? You may need to use `math.isclose()` if you're bothered about [performance](https://stackoverflow.com/a/33024979) or [unexpected behaviour like this](https://stackoverflow.com/questions/26091689/python-floating-point-number-comparison)

Reply:
`PositionAndAngle` parses the given numbers into float anyway so I don't think this actually changes anything. But you are right that we should be more conscious of performance in the server in general, the reflectometry server is probably not as responsive as it could be and this is an area for future improvement.

Reply:
OK, for now let's ignore it - we could probably start using more of numpy to improve performance and safety, but lets worry about that another time